Model/Classes/Flight.cs

```diff
@@ -0,0 +1,30 @@
+﻿using AirportTicketBooking.Utils;
+using System.Text;
+
+namespace AirportTicketBooking.Model.Classes;
+#nullable disable
+public class Flight
+{
+    private static long _id = 0;
+    public long Id { get; init; } = ++_id;
+    public Airport DepartureAirport { get; set; }
+    public Airport DestinationAirport { get; set; }
+    public Country DepartureCountry { get; set; }
+    public Country DestinationCountry { get; set; }
+    public DateTime Time { get; set; }
+    public FlightClass Class {  get; set; }
+
+
+    public override string ToString()
+    {
+        StringBuilder sb = new StringBuilder();
```

Comment:
Why `StringBuilder` is used? can we use $""? Here the code is not complicated and won't have a performance impact